libs/tracing/src/main/kotlin/net/corda/tracing/impl/BraveTracingService.kt

```diff
@@ -90,6 +85,25 @@ internal class BraveTracingService(serviceName: String, zipkinHost: String?, sam
         tracingBuilder.build().also(resourcesToClose::push)
     }
 
+    private fun sampler(samplesPerSecond: SampleRate): Sampler? = when (samplesPerSecond) {
```

Comment:
Let's make the return type non-nullable

```suggestion
    private fun sampler(samplesPerSecond: SampleRate): Sampler = when (samplesPerSecond) {
```